docs/concepts/capabilities.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: 'Capabilities'
+icon: 'ti-anchor' # themify icon pack : https://themify.me/themify-icons
+description: 'Application declaration, deployment, and management'
+sidebar_position: 1
+type: 'docs'
+---
+
+## Overview
+
+A ***capability** is a pluggable piece of functionality for an application. More specifically, capabilities are common, abstracted functionalities&mdash;usually the sort of utilities that aren't core business logic for a given application, such as...
+
+* HTTP Client & Server
+* Logging
+* Key-Value Storage
+* Messaging
+* Blob Storage
+* Secrets
+
+You can find a complete list of first-party capabilities in the [Capability Catalog](/docs/capabilities/index.mdx).
+
+## How capabilities work
+
+Capabilities are enacted by an **interface** and one or more swappable **capability providers**. A provider delivers a specific implementation of a given capability by communicating with other entities in the common tongue of a standard API. 
+
+For example...
+
+* The [Redis key-value storage provider](https://github.com/wasmCloud/wasmCloud/pkgs/container/keyvalue-redis) delivers the key-value capability to application components over the standard `wasi:keyvalue` API. 
```

Comment:
I don't know how it would work inside this bulleted list, but this is a great chance to show a *small* subset of the WIT for `wasi:keyvalue` -- to get a feel of the operations that they'd have access to

Reply:
@vados-cosmonic Took a stab at it -- what do you think?